docs/conf.py

```diff
@@ -41,8 +41,18 @@
     'autoapi.extension',
     'nbsphinx',
     'sphinx_last_updated_by_git',
+    'sphinx_design',
+    'sphinx-favicon',
 ]
 
+
+favicons = [
+   {
+      "rel": "icon",
+      "static-file": "favicon.svg",
```

Comment:
Did you make the favicon? Very fancy :)

Reply:
Ha ha, yes. "Graphic Design is my passion" 😝